cves/CVE-2004-0747.yml

```diff
@@ -119,10 +128,21 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: 58f17628acc2446d1811095b38fdcbb1357d6d55
+      note: |
+        This commit moved the ap_resolve_env function from server/util.c to server/core.c in preparation
+        of a new feature. Does not appear to be for security reasons, as the code written for the fix in 2004 was
+        unmodified
```

Comment:
It might be helpful to expand on why this is interesting. Maybe elaborate on what that new feature was?

Reply:
That's a good point. I could a little more contextual information about this change but the main point was that it was used as part of a new feature and wasn't modified, solidifying the probability that the fix was a good one. I'll add this to my commit.